website/content/docs/audit/index.mdx

```diff
@@ -29,17 +29,22 @@ The audit logs contain the full request and response objects for every
 interaction with Vault. The request and response can be matched utilizing a
 unique identifier assigned to each request.
 
-With a few specific exceptions, all strings (including authentication tokens and lease information) contained within requests and
-responses are hashed with a salt using HMAC-SHA256. The purpose of the hash is
-so that secrets aren't in plaintext within your audit logs. However, you're
-still able to check the value of secrets by generating HMACs yourself; this can
-be done with the audit device's hash function and salt by using the
-`/sys/audit-hash` API endpoint (see the documentation for more details).
+Most strings contained within requests and responses are hashed with a salt using HMAC-SHA256. The purpose of the hash is so that secrets aren't in plaintext within your audit logs. However, you're still able to check the value of secrets by generating HMACs yourself; this can be done with the audit device's hash function and salt by using the `/sys/audit-hash` API endpoint (see the documentation for more details).
 
 Note that currently only strings coming from JSON or being returned in JSON are
 HMAC'd. Other data types, like integers, booleans, and so on, are passed
 through in plaintext.
 
+While most strings are hashed, Vault does make some exceptions, such as auth and secrets, and users can enable additional exceptions using the [secrets enable](https://www.vaultproject.io/docs/commands/secrets/enable) command, and then tune it afterward.
```

Comment:
Don't we usually use relative links?  I would think Vercel previews wouldn't work correctly otherwise.

Reply:
I've seen cases where the docs use both direct and relative links but I went ahead and changed all the links to relative. BTW, I checked Vercel preview and it actually does work with direct links as well, just fyi.

Reply:
It "works" but it doesn't link to the preview data.  Let's say you moved a page in your changes to a new place.  The link would work because it refers to vaultproject.io, but if we deployed the changes, the link would be broken, because of the move.  This would be discoverable in the preview with a relative link, that points to the preview itself.

Reply:
Good to know, thanks!